universal-application-tool-0.0.1/app/services/WellKnownPaths.java

```diff
@@ -1,8 +1,8 @@
 package services;
 
 public class WellKnownPaths {
-  public static Path APPLICANT_FIRST_NAME = Path.create("applicant.name.first");
-  public static Path APPLICANT_MIDDLE_NAME = Path.create("applicant.name.middle");
-  public static Path APPLICANT_LAST_NAME = Path.create("applicant.name.last");
+  public static Path APPLICANT_FIRST_NAME = Path.create("applicant.name.first_name");
```

Comment:
question (non-blocking): Does changing this break anything? (previous submissions, etc.)

Reply:
It is currently broken and this is fixing it!  This needs to stay in sync with the scalar locations in `NameQuestionDefinition`.